x-pack/plugins/cases/common/schema/index.ts

```diff
@@ -55,12 +61,12 @@ export const limitedStringSchema = (fieldName: string, min: number, max: number)
     rt.identity
   );
 
-export const limitedArraySchema = <T extends rt.Mixed>(
-  codec: T,
-  min: number,
-  max: number,
-  fieldName: string
-) =>
+export const limitedArraySchema = <T extends rt.Mixed>({
+  codec,
+  fieldName,
+  min,
+  max,
+}: { codec: T } & LimitedStringSchemaType) =>
```

Comment:
super nit: `LimitedStringSchemaType` doesn't seem fitting.

Just thought it was weird that **String** was in the name, maybe `LimitedSchemaType` would be more appropriate.